config/etc/zinit/haveged.yaml

```diff
@@ -0,0 +1,2 @@
+exec: haveged -w 1024 -d 32 -i 32 -v 1
+oneshot: true
```

Comment:
This should be removed, it's daemon so remove oneshot to get restarted if crached

Reply:
I'm not sure, I didn't find a way to make not make it fork and stay in the foreground